Automattic/sensei
# Add existing lesson

Resolves #3716

We allow to add only courseless lessons.


## Proposed Changes
* Add an option to add an existing lesson in the Course Outline block. It works only with lessons that are not attached to a course yet.

## Testing Instructions

1. Create a course.
2. In the course outline, click on Add (+) button. Choose "Existing Lesson(s)".
3. In the filter view choose lesson or a few.
4. Click Add selected.
5. It is expected to see added lessons in the updated course outline.
6. Save the course.
7. Try to look at the course on the frontend.
8. All added lessons should be there.

## Pre-Merge Checklist
